pkgs/development/python-modules/tree-sitter-language-pack/default.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  fetchPypi,
+  cython,
+  setuptools,
+  typing-extensions,
+  tree-sitter,
+  tree-sitter-c-sharp,
+  tree-sitter-embedded-template,
+  tree-sitter-yaml,
+}:
+
+let
+  pname = "tree_sitter_language_pack";
```

Comment:
Probably not a good idea to name this variable `pname` when it is, in fact, not the pname for this.

Reply:
Reasonable suggestion, I have remove it.